src/screens/SearchResult.js

```diff
@@ -32,9 +30,16 @@ const SearchResult = () => {
       </Grid>
     </Grid>
   );
+  useEffect(() => {
```

Comment:
made changes for Api calling in a callback to resolve the 304 code. Now with this implementation, the API gets 304 status only once or none after getting the response.